src/dropdown/list/dropdown-list.component.ts

```diff
@@ -519,7 +520,7 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 				}
 			} else if (event.key === "ArrowUp") {
 				if (this.hasPrevElement()) {
-					this.getPrevElement().scrollIntoView({  block: "nearest" });
+					this.getPrevElement()?.scrollIntoView({  block: "nearest" });
```

Comment:
Based on the change you made to `getNextElement()`, shouldn't optional chaining be used on line 517 instead of 523?

Reply:
Hi @Akshat55 
Yes, it went unnoticed. Fixed it now.